src/lib.rs

```diff
@@ -1,7 +1,7 @@
 pub mod consensus;
 pub mod decision;
 pub mod fault;
-pub mod mvba;
+// pub mod mvba;
```

Comment:
@b00f I've commented out `mvba` since it has clippy failures (clippy failures prevent releases from going through)

Reply:
I  fixed it in #72